src/main/java/com/example/planit/utill/Constants.java

```diff
@@ -10,7 +10,7 @@ public class Constants {
 
     public static final long MINUTES_TO_MILLIS = 60000;
 
-    public static final int HOUR_IN_MINUTES = 60;
+    public static final int MINUTES_IN_SECONEDS = 60;
 
```

Comment:
You have a typo in "SECONDS" (you wrote "SECONEDS").
Also, ` MINUTE_IN_SECONEDS` is not so readable. Because the word "in" should not be used like that ( it sounds like you have a minute inside the seconds. At first I didn't understand what this constant represents.
I think you meant "SECONDS_IN_A_MINUTE", or "MINUTE_AS_SECONDS".